src/validator/erc20/validator.js

```diff
@@ -0,0 +1,75 @@
+const localList = require("./erc20-list.json");
+const axios = require("axios");
+// const tlist = require("../../../dist/v0.0.21/1/tokens.json");
+
+const getErc20List = async () => {
+  await axios
+    .get("https://api.coingecko.com/api/v3/coins/list")
+    .then((res) => {
+      console.log(res);
+    })
+    .catch((e) => {
+      console.log(e);
+    });
+};
+
+/**
+ * @param { Array<{name:string, id: string, symbol:string, address: string}> } tokenList
+ */
+const validator = async (tokenList) => {
+  // const erc20List = (await getErc20List()) || localList;
+  const erc20List = localList.data;
+  let res = 1;
+  const errArr = [];
+  for (let i = 0; i < tokenList.length; i++) {
+    const token = tokenList[i];
+    const isInList = erc20List.filter((item) => {
```

Comment:
Create a set first and then check the existence in that.

Reply:
modified